scimath/units/unit_scalar.py

```diff
@@ -17,5 +17,9 @@ class UnitScalar(UnitArray):
         (UnitScalar(25, units='0.0001*m**2'), 0.0001*m**2)
     '''
     def __repr__(self):
-        return ("UnitScalar(%s, units='%s')" 
+        return ("UnitScalar(%s, units='%s')"
                 % (self.item().__repr__(), repr(self.units)))
+
+    def __str__(self):
+        return ("UnitScalar(%s, units='%s')"
+                % (self.item().__str__(), self.units.label))
```

Comment:
Thanks for the comment @rkern but I don't understand it:
1. I am doing here what I was doing for arrays
2. The output for something like `UnitScalar(1, units="cm")` can be `eval()`-ed because "cm" is parsed. If people create their own units, and add them to their parser, it will be parsed.
3. I always thought that `str` wasn't required to return something that could be `eval()`-ed and was just about returning something readable.

Sorry if I am missing something obvious.